UnityProject/Assets/Scripts/Sprite Handler/SpriteHandler.cs

```diff
@@ -45,6 +45,7 @@ public class SpriteHandler : MonoBehaviour
 
 	private bool subCatalogueChanged = false;
```

Comment:
`isSubCatalogueChanged`?

Reply:
I feel like this is another case of ``is`` not being a good representation of what the variable represents.